samples/Azure.Management.Storage/Generated/BlobService.cs

```diff
@@ -86,7 +86,7 @@ internal static void ValidateResourceId(ResourceIdentifier id)
         /// <returns> An object representing collection of BlobContainers and their operations over a BlobContainer. </returns>
         public virtual BlobContainerCollection GetBlobContainers()
         {
-            return new BlobContainerCollection(Client, Id);
+            return GetCachedClient((Client) => new BlobContainerCollection(Client, Id));
```

Comment:
I am curious why the simplifier does not automatically remove this extra parenthesis - could we remove it in our writer? 
```suggestion
            return GetCachedClient(Client => new BlobContainerCollection(Client, Id));
```